plugin.json

```diff
@@ -16,7 +16,7 @@
             "de": "Wayfair plugin fur Plentymarkets Kunden"
       },
       "serviceProvider": "Wayfair\\Providers\\WayfairServiceProvider",
-      "isClosedSource": true,
+      "isClosedSource": false,
```

Comment:
Nice, this is what controls this: 
![image](https://user-images.githubusercontent.com/22055435/96181469-0c8f4800-0f02-11eb-80eb-0410cb0408bc.png)

Reply:
yup!